src/content/RepoPage/RepoPage.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const RepoPage = () => <div>REPO PAGE</div>;
```

Comment:
This RepoPage const should be replaced with `const RepoPage = () => {
  return <div>REPO PAGE</div>;
};`

See https://www.carbondesignsystem.com/tutorial/react-step-1#import-and-export-content-pages